src/parser/mod.rs

```diff
@@ -136,7 +135,7 @@ mod tests {
             ("1e6", 1e6),
             ("-2E-2", -2E-2),
             ("6.626e-34", 6.626e-34),
-            ("9_224_617.445_991_228_313", 9_224_617.445_991_228_313),
+            ("9_224_617.445_991_228_313", 9_224_617.445_991_227),
```

Comment:
```
f64
float has excessive precision
`#[deny(clippy::excessive_precision)]` implied by `#[deny(warnings)]`
for further information visit https://rust-lang.github.io/rust-clippy/master/index.html#excessive_precisionclippyexcessive_precision
```
Although I only get the warning in vscode with rust analyzer running `cargo clippy` doesn't give any errors/warnings?

Reply:
Hmm, could you merge master to run CI, it should include `cargo clippy` pass?

Reply:
By merge master do you mean my own branch or just push here and run CI?

Reply:
Thanks, I meant merging origin/master to your branch to include https://github.com/ordian/toml_edit/commit/cd6d1fb2cf43e5472d7eac8fb707bebaf283d9ac. Now it's working.

I think in this case, it's rust-analyzer's fault, because the type here is `f64`, not `f32`, so there's no excessive precision here.

Reply:
Could you please revert this change?

Reply:
Ahh, it's because clippy doesn't run for tests unless you `--all-targets`. I did change it back to longer decimal, do you want me to add the `--all-targets` to CI or just leave it?

Reply:
I see, thanks for digging into this. Yes, please add it here
https://github.com/ordian/toml_edit/blob/cd6d1fb2cf43e5472d7eac8fb707bebaf283d9ac/.github/workflows/ci.yml#L72